net/ooniprobe/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ooniprobe
+PKG_VERSION:=3.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=probe-cli-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ooni/probe-cli/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=d1e8c376d89bd7aa3a0971eac95abf1bbd438df072f7f9e5b9e32d8753ff3f71
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/probe-cli-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host measurement-kit
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/ooni/probe-cli
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/ooniprobe
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=OONI probe-cli
+  URL:=https://ooni.org
+  DEPENDS:=\
+	$(GO_ARCH_DEPENDS) +libevent2-openssl \
+	+libevent2-core +libevent2-extra +libevent2-pthreads \
+	+libmaxminddb +libnghttp2 +libopenssl \
+	+zlib +libcurl +libstdcpp
```

Comment:
This is a go package that needs libstdcpp?

Reply:
I think that reason for that is, that measurement_kit lib is statically linked to ooniprobe